test.js

```diff
@@ -14,3 +14,4 @@ require('./test/resampler')
 require('./test/stream')
 require('./test/log')
 require('./test/rational')
+require('./test/io-context-streaming')
```

Comment:
I think that we should have rather a

```suggestion
require('./test/io-context')
```